chromadb/test/property/invariants.py

```diff
@@ -157,3 +157,7 @@ def ann_accuracy(
         pass  # it's ok if we're running outside hypothesis
 
     assert recall >= min_recall
+
+    # Ensure that the query results are sorted by distance
+    for distance_result in query_results["distances"]:
+        assert np.allclose(np.sort(distance_result), distance_result)
```

Comment:
O(n + n log n) time complexity, could be done in O(n)

Not sure if we care about that enough to make the code more complex. It's strange to me that python has no `nondecreasing` built-in.

Reply:
yeah thought about this, i don't think its a big deal as N is pretty small here and wanted to keep the code clean